src/Parser/Priest/Holy/Modules/PriestCore/HolyWordsReductionBySpell.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import ExpandableStatisticBox from 'Interface/Others/ExpandableStatisticBox';
+import { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+
+import HolyWords from './HolyWords';
+
+
+class HolyWordsReductionBySpell extends Analyzer {
+  static dependencies = {
+    holyWords: HolyWords,
+  };
+
+  totalReductionDuration = 0;
+
+  get totalReduction () {
+    this.totalReductionDuration = Object.values(this.holyWords.reductionAmountBySpell).map(obj => obj.amount).reduce((total, casts) => total + casts, 0);
+    return this.totalReductionDuration;
+  }
+
+
+  statistic() {
+    return (
+      <ExpandableStatisticBox
+        position={STATISTIC_ORDER.CORE(6)}
+        icon={<SpellIcon id={SPELLS.HOLY_WORDS.id} />}
+        value={`${this.totalReduction}  seconds`}
```

Comment:
Generally speaking it'd be nice to normalize this by fight length, showing it as a CDR % (see the Brewmaster brew CDR code for an example) so that consistent performance looks consistent across fights.